suzieq/engines/pandas/device.py

```diff
@@ -110,25 +108,20 @@ def get(self, **kwargs):
         # The poller merge kills the filtering we did earlier, so redo:
```

Comment:
In line 106 we are returning the dataframe without stripping off the columns to drop, so we show the empty dataframe with the internal columns. Moreover, should we move this check before merging the device df with the poller one? (so before line 56)

Reply:
Nice try, but no dice :) The model split is a little more complicated because model names can have spaces inside them. So the split happens a little above, if you read it more carefully :-p

Reply:
I fixed it to return the fields only requested. However, I left this logic where it is because its possible that we end up with an empty dataframe which might happen if there's no poller info. Should we return the device info even without the poller info? Not sure I agree. I also took the time to clean up some old cruft in deciding what fields to return. Did a git push -f